DataMerge.ipynb

```diff
@@ -0,0 +1,180 @@
+{
```

Comment:
<p>Maybe add a graph colored by node population here to visualize this (check out <code>ox.get_node_colors_by_attr</code>)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/laitalaj/railanalysis/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>